src/WpfMath/WpfMath.csproj

```diff
@@ -1,7 +1,9 @@
-﻿<Project Sdk="Sunburst.NET.Sdk.WPF.Patched/1.0.49">
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
     <PropertyGroup>
-        <TargetFramework>net452</TargetFramework>
-        <LangVersion>7.2</LangVersion>
+        <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
```

Comment:
Could you please elaborate: why doesn't `netcoreapp3.0;net452` work for you? I've checked and it just seems to work.

May it be that you're confusing multi-targeting and `netstandard2.0`? `net461` would really be a minimal requirement if we target .NET Standard, but in this PR we're still multi-targeting .NET Core 3.0 and .NET Framework (which is good!).

Reply:
If you target net452, it will cause the error:
```
C:\Program Files\dotnet\sdk\3.0.100\Sdks\Microsoft.NET.Sdk.WindowsDesktop\targets\Microsoft.WinFX.targets(243,9): error MC1000: Unknown build error, 'Could not find assembly 'System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e'. Either explicitly load this assembly using a method such as LoadFromAssemblyPath() or use a MetadataAssemblyResolver that returns a valid assembly.'  [C:\Users\hez20\source\repos\wpf-math\src\WpfMath\WpfMath.csproj]
```

Because .NET Framework 4.5.2 SDK doesn't contain assembly `System.Core`.

Reply:
I can reproduce that on WPF-Math but cannot reproduce on a new WPF project. Will investigate.

Reply:
`System.ValueTuple` package reference creates the trouble here. You cannot build even a simple WPF app with this package even if you don't use multitargeting. I'll report the issue to .NET SDK repository.

Reply:
Here it is: https://github.com/dotnet/sdk/issues/3770

Reply:
Let's get rid of tuples in our code for now. There aren't too many of them.